api_app/serializers.py

```diff
@@ -592,3 +597,74 @@ class EnvelopeSerializer(rfs.Serializer):
         results = serializer_class(many=many)
 
     return EnvelopeSerializer
+
+
+class CustomConfigSerializer(rfs.ModelSerializer):
+    owner = PrimaryKeyRelatedField(queryset=User.objects.all(), required=False)
+
+    def validate_value(self, value):
+        try:
+            json.loads(value)
+        except json.JSONDecodeError:
+            raise ValidationError("Value is not JSON-compliant.")
+        return value
```

Comment:
why is this necessary?

Reply:
The same explanation as the `json.loads` one